backends/dpdk/README.md

```diff
@@ -74,6 +74,13 @@ In P4 the second argument of the extract method is the number of bits.
 Compiler generates instructions which compute the size in bytes from the value in bits.
 If the value in bits is not a multiple of 8, the value is rounded down to the lower
 multiple of 8 bits.
+- Currently dpdk target does not support standard count and execute methods for Counter and Meter externs as defined in PSA and PNA specifications. It requires packet length as parameter in count and execute methods.
+```Meter
```

Comment:
Does DPDK development have any plans in the future to support the standard count and execute methods, without a packet length parameter?

Reply:
There is no plan in the near future. There was an earlier discussion of providing ethernet frame size as default packet length, however it seems that it wouldn't be correct as DPDK implements trCM which do not work on ethernet frame length.

Reply:
"which do not work on ethernet frame length" Meters/policers can work on whatever length in bytes you give for a packet, whether it matches the packet's length or not, so it isn't clear to me what your statement means.  If DPDK implements meters by default with ethernet frame length, then I would recommend enabling them to do so, and DOCUMENTING that the methods that do not take a packet length as input, use the ethernet frame length in bytes.  If they work as documented, then I'd say that they work :-)

Reply:
I will take this up with the DPDK target team again and see if they can have this in their worklist. We can enable the p4c support when we have the default packet length available from the target side.